xmppserver/src/main/java/org/jivesoftware/openfire/net/ServerSocketReader.java

```diff
@@ -47,7 +47,9 @@
  * by changing the property <b>xmpp.server.processing.max.threads</b>.
  *
  * @author Gaston Dombiak
+ * @deprecated see {@link ServerStanzaHandler}
  */
+@Deprecated
```

Comment:
This entire class is probably eligble to be removed under OF-2635 right?

Reply:
Yes. Feel free to remove!